vignettes/covariance.Rmd

```diff
@@ -200,4 +200,31 @@ homogeneous compound symmetry covariance structure with total only $k=2$
 variance parameters, otherwise we obtain the heterogeneous compound
 symmetry covariance structure with $k = m + 1$ variance parameters.
 
+# Spatial Covariance Structure
```

Comment:
Can we rename to "Euclidean" ?

Reply:
"Covariance structures induced by covariance functions" ?

"Gaussian process covariance structures"?

special case: stationary covariance functions based on Euclidean distance. I guess the best name depends a bit on whether there is any chance that we might want to support more covariance functions.

Reply:
I think we keep it to Euclidean for the time being. Anyway I don't see large demand for these.